### DISCUSSION TITLE
新版本阿里云盘没有视频转码功能了吗

### DISCUSSION THREAD

**Discussion by moesnow** [AlistGo/alist#215] [Q&A] [answered]
看了下文档没找到

我记得旧版本有个原画/转码切换开关，safari好多视频原画看不了就靠转码功能了
Macos还能用iina，iPhone/iPad不行啊

**Comment by xhofe** [comment, marked as answer]
他那个转码后的链接 有效期只有15分钟，感觉有点鸡肋。但是保留了接口，https://alist-doc.nn.ci/docs/api#publicpreview 。

**Reply by holll** [reply]
这个接口或许有用，对于转码的视频，这个接口返回的是m3u8的链接，比较适合在线观看。我的想法是，能不能在视频播放页面增加一个按钮，可以尝试使用转码后的链接播放。
